src/content/docs/alerts-applied-intelligence/overview.mdx

```diff
@@ -70,7 +70,7 @@ Use the correlate features to define your data sources and to review and configu
 * **[Sources](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/get-started-incident-intelligence/#1-configure-sources)**: Manage the data input sources you've chosen to analyze and be notified about. You can add new sources or configure existing ones.
 * **[Decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions/)**: Review your correlated incidents and make decisions on how to respond to them. You can also review, edit, and add decisions.
 
-## **Enrich and notify** your notifications with useful metadata [#enrich]
+## **Enrich** your **notifications** with useful metadata [#enrich]
```

Comment:
Why are we bolding a title? I have nothing against it, I've just never seen it before. Feel free to ignore and merge though!

Reply:
Trying to make a stronger connection between these words and what's the UI. Consider it an experiment